sdk/core/azure-core/test/ut/service_tracing_test.cpp

```diff
@@ -21,14 +21,14 @@ TEST(TracingContextFactory, ServiceTraceEnums)
     spanKind = SpanKind::Producer;
     spanKind = Azure::Core::Tracing::_internal::SpanKind::Server;
     int i = static_cast<int>(spanKind);
-    i += 1;
+    (void)i; // silence clang 14 warning about unused variable i.
   }
   {
     SpanStatus spanStatus = SpanStatus::Unset;
     spanStatus = SpanStatus::Error;
     spanStatus = SpanStatus::Ok;
     int i = static_cast<int>(spanStatus);
```

Comment:
What is going on with `i`? Here?
We declare it, but never use it, and also the scope below redeclares it?

Reply:
The intent is basically the same as `std::underlying_type<T>`. 

This was the minimal change I could come up with which silenced the compiler error with clang 14 so I could test my code on Linux.